inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java

```diff
@@ -50,6 +51,7 @@
  * Use to cache the sort cluster config and reduce the number of query to database.
  */
 @Service
+@Lazy
```

Comment:
Is it possible to put `@Lazy` on the line above `@Service` or `@Autowired`?

- This makes the code format look better.

Reply:
fixed, thx